CHANGELOG.md

```diff
@@ -3,11 +3,14 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 Use the following format for additions: ` - VERSION: [feature/patch (if applicable)] Short description of change. Links to relevant issues/PRs.`
 
-- 1.4.9:
-  - handle failed promises [#1017](https://github.com/FredrikNoren/ungit/issues/1017)
-  - empty commit [#1028](https://github.com/FredrikNoren/ungit/issues/1028)
-  - fix commit detail layout while hovering over commit node [#1025](https://github.com/FredrikNoren/ungit/issues/1025)
-- 1.4.8: fix remote branches display name and delete action [#1032](https://github.com/FredrikNoren/ungit/issues/1032), [#1031](https://github.com/FredrikNoren/ungit/issues/1031)
+- 1.4.10:
+  - differentiate remote vs local tag. [#1016](https://github.com/FredrikNoren/ungit/issues/1016)
+  - fix push not throwing giterror
+  - fix remote tag push not creating remote tag
+  - change ref refresh logic
+  - show error on incorrect credentials [#1042](https://github.com/FredrikNoren/ungit/pull/1042)
+  - allow credential handling for remotes [#1041](https://github.com/FredrikNoren/ungit/pull/1041)
```

Comment:
Nit: We always link to the issues instead of the PR if one exists.

#1042 > https://github.com/FredrikNoren/ungit/issues/1039
#1036 > https://github.com/FredrikNoren/ungit/issues/1029